frontend/src/components/common/LogRecordsTable.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import {
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@chakra-ui/react";
+import { LogRecord } from "../../types/LogRecordTypes";
+
+type Props = {
+  logRecords: LogRecord[];
+};
+
+const LogRecordsTable = ({ logRecords }: Props): React.ReactElement => {
+  return (
+    <div>
+      <TableContainer marginTop="12px" height="70vh" overflowY="scroll">
+        <Table
+          variant="simple"
+          style={{ minHeight: "400px", verticalAlign: "middle" }}
+        >
+          <Thead className="table-header">
```

Comment:
the width of the headers seem to change when flipping through pages, looks like the length of the resident name is increasing the width.
![image](https://github.com/uwblueprint/supportive-housing/assets/57375371/ba4e3c38-d567-4495-9072-c60e9a3e03b0)
![image](https://github.com/uwblueprint/supportive-housing/assets/57375371/a91405aa-0761-4603-a210-93a9ac2c9094)

Reply:
This issue will be dealt with in https://github.com/uwblueprint/supportive-housing/issues/97 ; resolving